### DISCUSSION TITLE
Automatic long text break to adjust scene width

### DISCUSSION THREAD

**Discussion by grivasgervilla** [ManimCommunity/manim#2818] [Suggestions and Proposals]
## Description of proposed feature
Attribute to set text width relative to scene width.

## How can the new feature be used?

I would like to do something like add `width = \scenewidth`, so long texts break automatically in several lines, maybe with proper hyphens, like it occurs in LaTeX where text is broken automatically.

## Additional comments

I do not know whether there is a way to make it with the current options.

**Comment by naveen521kk** [comment]
It should be possible with https://github.com/ManimCommunity/manim/issues/2355 for `Text` API's, for LaTeX I'm not sure.

**Comment by grivasgervilla** [comment]
> It should be possible with #2355 for `Text` API's, for LaTeX I'm not sure.

Thanks for your response. By that commit, I understand that this is not available yet. In addition, I think that this behaviour should be provided by default in each text object (LaTeX or non-LaTeX), since it will ease the elaboration of new animations with long texts. I do not know how other software obtain this, but I understand that since LaTeX and, for example, Markdown processor are capable of break text to adjust the document width it could be done.

Maybe the `needs discussion` label should be added to this commit.

Many thanks for your time and this awesome project.

**Comment by naveen521kk** [comment]
I think it should be possible for LaTeX also, something should be done when invoking `dvisvgm`, not sure should look. Also, https://github.com/ManimCommunity/manim/issues/1390 needs to be fixed before this can be implmented.
